notebooks/cookbook/report_minimization.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e174828a",
+   "metadata": {},
+   "source": [
+    "## Reporting Minimization\n",
+    "\n",
+    "You can report the status of [simulation.minimizeEnergy()](http://docs.openmm.org/latest/api-python/generated/openmm.app.simulation.Simulation.html?#openmm.app.simulation.Simulation.minimizeEnergy) using a [MinimizationReporter](http://docs.openmm.org/development/api-python/latest/openmm.openmm.MinimizationReporter.html). Note that `MinimizationReporter` was introduced in OpenMM 8.1.\n",
```

Comment:
The URL for MinimizationReporter is broken.  It should be http://docs.openmm.org/latest/api-python/generated/openmm.openmm.MinimizationReporter.html.

Reply:
Thanks for spotting that!